test/clt-tests/sharding/syntax/replication-test.rec

```diff
@@ -0,0 +1,96 @@
+––– block: ../base/init –––
+––– input –––
+export INSTANCE=1
+––– output –––
+––– block: ../base/flexible/start-searchd –––
+––– input –––
+export INSTANCE=2
+––– output –––
+––– block: ../base/flexible/start-searchd –––
+––– input –––
+export INSTANCE=3
+––– output –––
+––– block: ../base/flexible/start-searchd –––
+––– input –––
+export INSTANCE=4
+––– output –––
+––– block: ../base/flexible/start-searchd –––
+––– input –––
+export INSTANCE=5
+––– output –––
+––– block: ../base/flexible/start-searchd –––
+––– input –––
+export CLUSTER_NAME=replication
+––– output –––
+––– block: ../base/flexible/create-cluster –––
+––– input –––
+export CLUSTER_NAME=replication
+––– output –––
+––– block: ../base/flexible/join-cluster –––
+––– input –––
+mysql -h0 -P1306 -e "CREATE TABLE testrt (title text, content text, gid uint);"; echo $?
+––– output –––
+0
+––– input –––
+mysql -h0 -P1306 -e "INSERT INTO testrt VALUES(0,'List of HP business laptops','Elitebook Probook',10);"; echo $?
+––– output –––
+0
+––– input –––
+mysql -h0 -P1306 -e "INSERT INTO testrt VALUES(0,'List of Dell business laptops','Latitude Precision Vostro',10);"; echo $?
+––– output –––
+0
+––– input –––
+mysql -h0 -P1306 -e "ALTER CLUSTER replication ADD testrt;"; echo $?
+––– output –––
+0
+––– input –––
+mysql -h0 -P1306 -e "SHOW STATUS LIKE 'cluster_replication_indexes';"
+––– output –––
++-----------------------------+--------+
+| Counter                     | Value  |
++-----------------------------+--------+
+| cluster_replication_indexes | testrt |
++-----------------------------+--------+
+––– input –––
+mysql -h0 -P1306 -e "SELECT * FROM testrt\G;"
+––– output –––
+*************************** 1. row ***************************
+id: %{NUMBER}
+title: List of Dell business laptops
+content: Latitude Precision Vostro
+gid: 10
+*************************** 2. row ***************************
+id: %{NUMBER}
+title: List of HP business laptops
+content: Elitebook Probook
+gid: 10
+––– input –––
+export INSTANCE=5
+––– output –––
+––– block: ../base/flexible/stop-searchd –––
+––– input –––
+mysql -h0 -P1306 -e "INSERT INTO replication:testrt VALUES(0,'List of HP business laptops','Macbook',10);"; echo $?
+––– output –––
+0
+––– input –––
+export INSTANCE=5
+––– output –––
+––– block: ../base/flexible/start-searchd –––
+––– input –––
+sleep 5; mysql -h0 -P5306 -e "SELECT * FROM testrt\G;"
+––– output –––
+*************************** 1. row ***************************
+id: %{NUMBER}
+title: List of Dell business laptops
+content: Latitude Precision Vostro
+gid: 10
+*************************** 2. row ***************************
+id: %{NUMBER}
+title: List of HP business laptops
+content: Macbook
+gid: 10
+*************************** 3. row ***************************
+id: %{NUMBER}
+title: List of HP business laptops
+content: Elitebook Probook
+gid: 10
```

Comment:
Before this you can test more:
* insert into the replicated table on non-1306
* make sure the document is visible on non-1306 and non-5306
* delete a document on 4306
* make sure it's deleted on 3306

Currently you add 5 nodes, but work only with 2 of them.